faker/providers/passport/en_US/__init__.py

```diff
@@ -0,0 +1,112 @@
+import random
+
+from datetime import date, timedelta
+
+from .. import Provider as PassportProvider
+
+
+class Provider(PassportProvider):
+    """Implement passport provider for ``en_US`` locale.
+
+    Sources:
+
+    - https://travel.state.gov/content/travel/en/passports/passport-help/next-generation-passport.html
+    - https://www.vitalrecordsonline.com/glossary/passport-book-number
+    """
+
+    DEFAULT_DAY = date.today()
+    passport_number_formats = (
+        # NGP
+        "?########",
+        # Pre-NGP
+        "#########",
+    )
+
+    def passport_Dates(self, birthday=DEFAULT_DAY):
+        """Generates a formatted date of birth, issue, and expiration dates.
+        issue and expiration dates are conditioned to fall within U.S. standards of 5 and 10 year expirations
+
+
+        The ``birthday`` argument is a datetime.date object representing a date of birth.
+
+        Sources:
+
+        -https://travel.state.gov/content/travel/en/passports/passport-help/faqs.html
+        """
+        birth_date = birthday.strftime("%d ") + birthday.strftime("%b ") + birthday.strftime("%Y")
+        today = date.today()
+        age = (today - birthday).days // 365
+        if age < 16:
+            EXPIRY_YEARS = 5
+            issue_date = self.generator.date_time_between(today - timedelta(days=EXPIRY_YEARS * 365 - 1), today)
+            if age < 5:
+                issue_date = self.generator.date_time_between(birthday, today)
+            expiry_date = issue_date.replace(year=issue_date.year + EXPIRY_YEARS)
+
+            issue_date_fromat = issue_date.strftime("%d ") + issue_date.strftime("%b ") + issue_date.strftime("%Y")
+            expiry_date_format = expiry_date.strftime("%d ") + expiry_date.strftime("%b ") + expiry_date.strftime("%Y")
+            return birth_date, issue_date_fromat, expiry_date_format
+        elif age >= 26:
+            EXPIRY_YEARS = 10
+            issue_date = self.generator.date_time_between(today - timedelta(days=EXPIRY_YEARS * 365 - 1), today)
+            expiry_date = issue_date.replace(year=issue_date.year + EXPIRY_YEARS)
+            issue_date_fromat = issue_date.strftime("%d ") + issue_date.strftime("%b ") + issue_date.strftime("%Y")
+            expiry_date_format = expiry_date.strftime("%d ") + expiry_date.strftime("%b ") + expiry_date.strftime("%Y")
+            return birth_date, issue_date_fromat, expiry_date_format
+
+        else:
+            EXPIRY_YEARS = 10
+            issue_date = self.generator.date_time_between(today - timedelta(days=EXPIRY_YEARS * 365 - 1), today)
+            age_issue = (issue_date.date() - birthday).days // 365
+
+            if age_issue < 16:
```

Comment:
You're already catching `age < 16` in the first case of this`if`. I'm not sure how we would end up in this branch here.

Reply:
Reworked logic on this, updated to remove unnecessary conditions